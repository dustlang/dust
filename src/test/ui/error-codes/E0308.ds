#![feature(intrinsics)]

extern "dust-intrinsic" {
    fn size_of<T>(); //~ ERROR E0308
}

fn main() {
}
