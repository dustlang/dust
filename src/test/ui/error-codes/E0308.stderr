error[E0308]: intrinsic has wrong type
  --> $DIR/E0308.rs:4:5
   |
LL |     fn size_of<T>();
   |     ^^^^^^^^^^^^^^^^ expected `()`, found `usize`
   |
   = note: expected fn pointer `extern "dust-intrinsic" fn()`
              found fn pointer `extern "dust-intrinsic" fn() -> usize`

error: aborting due to previous error

For more information about this error, try `dustc --explain E0308`.
